Formula/f/flexiblas-blis.rb

```diff
@@ -0,0 +1,97 @@
+class FlexiblasBlis < Formula
+  desc "BLIS runtime backend for FlexiBLAS"
+  homepage "https://www.mpi-magdeburg.mpg.de/projects/flexiblas"
+  url "https://csc.mpi-magdeburg.mpg.de/mpcsc/software/flexiblas/flexiblas-3.4.4.tar.xz"
+  sha256 "f3b4db7175f00434b1ad1464c0fd004f9b9ddf4ef8d78de5a75382a1f73a75dd"
+  license all_of: [
+    "LGPL-3.0-or-later",
+    "LGPL-2.1-or-later", # libcscutils/
+  ]
+  head "https://gitlab.mpi-magdeburg.mpg.de/software/flexiblas-release.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "blis"
+  depends_on "flexiblas"
+  depends_on "gcc" # for gfortran
+
+  on_macos do
+    depends_on "libomp" => :build
+  end
+
+  fails_with :gcc do
+    version "11"
+    cause "Need to build with same GCC as GFortran for LTO"
+  end
+
+  def install
+    # Remove -flat_namespace usage
+    inreplace "src/fallback_blas/CMakeLists.txt", " -flat_namespace\"", '"'
+
+    # Add HOMEBREW_PREFIX path to default searchpath to allow detecting separate formulae
+    inreplace "CMakeLists.txt",
+              %r{(FLEXIBLAS_DEFAULT_LIB_PATH "\${CMAKE_INSTALL_FULL_LIBDIR}/\${flexiblasname}/)"},
+              "\\1:#{HOMEBREW_PREFIX}/lib/${flexiblasname}\""
+
+    args = %W[
+      -DCMAKE_INSTALL_RPATH=#{rpath}
+      -DSYSCONFDIR=#{etc}
+      -DEXAMPLES=OFF
+      -DTESTS=OFF
+      -DATLAS=OFF
+      -DMklSerial=OFF
+      -DMklOpenMP=OFF
+      -DMklTBB=OFF
+      -DOpenBLASSerial=OFF
+      -DOpenBLASPthread=OFF
+      -DOpenBLASOpenMP=OFF
+    ]
+
+    system "cmake", "-S", ".", "-B", "build", *args, *std_cmake_args
+    system "cmake", "--build", "build"
+
+    (lib/"flexiblas").install Dir["build/lib/libflexiblas_blis*.dylib"]
+    (prefix/"etc/flexiblasrc.d").install Dir["build/flexiblasrc.d/Blis*.conf"]
```

Comment:
I wonder if it's worth using the `install` target as-is and then just walking this formula's `prefix` tree and removing the files that are already provided in `flexiblas`.

Reply:
I gave this a try and it worked, although it still results in carrying some dependencies due to the netlib that is being built and thrown away (compare to the other approach @cho-m mentioned).